docs/pages/management/dynamic-resources/user-and-role.mdx

```diff
@@ -0,0 +1,501 @@
+```
+
+List the created Kubernetes resources:
+
+```code
+$ kubectl get teleportroles -n "$OPERATOR_NAMESPACE"
+# NAME       AGE
+# engineer   10m
+# manager    10m
+
+$ kubectl get teleportusers -n "$OPERATOR_NAMESPACE"
+# NAME     AGE
+# alice    10m
+# bob      10m
+```
+
+</TabItem>
+<TabItem label="terraform">
+
+```code
+$ terraform plan
+[...]
+Plan: 4 to add, 0 to change, 0 to destroy.
+
+$ terraform apply
+teleport_role.engineer: Creating...
+teleport_role.manager: Creating...
+teleport_role.engineer: Creation complete after 0s [id=engineer]
+teleport_role.manager: Creation complete after 0s [id=manager]
+teleport_user.bob: Creating...
+teleport_user.alice: Creating...
+teleport_user.bob: Creation complete after 0s [id=bob]
+teleport_user.alice: Creation complete after 0s [id=alice]
+
+Apply complete! Resources: 4 added, 0 changed, 0 destroyed.
+```
+
+</TabItem>
+</Tabs>
+
+### Step 3/4 - Validate users were created
+
+Now that the IaC tooling has run, we'll validate that the users were properly
+created and granted the correct roles.
+
+<Tabs>
+<TabItem label="via the UI">
+
+If you have UI access, connect to your Teleport cluster Web UI, open the
+management panel, and select the "Users" tab.
+
+![Screenshot of the web UI listing alice and bob users](../../../img/management/check-users-web-ui.png)
+
+Two new users `alice` and `bob` should be present.
+
+</TabItem>
+<TabItem label="via CLI">
+
+```code
+$ tctl users ls
+User                          Roles
+----------------------------- -------------------------
+@teleport-access-approval-bot @teleport-access-approver
+alice                         manager,engineer
+bob                           engineer
+bot-operator                  bot-operator
+
+# alternatively you can inspect the users details by doing
+$ tctl get user/alice
+kind: user
+metadata:
+  id: 1704849160091933780
+  labels:
+    teleport.dev/origin: kubernetes
+  name: alice
+spec:
+  created_by:
+    time: "2024-01-10T01:12:40.088581806Z"
+    user:
+      name: bot-operator
+  expires: "0001-01-01T00:00:00Z"
+  roles:
+  - manager
+  - engineer
+  status:
+    is_locked: false
+    lock_expires: "0001-01-01T00:00:00Z"
+    locked_time: "0001-01-01T00:00:00Z"
+    recovery_attempt_lock_expires: "0001-01-01T00:00:00Z"
+version: v2
+```
+
+</TabItem>
+</Tabs>
+
+### Step 4/4 - Create a password reset link
+
+At this point, the local users have been created in Teleport. However, we never
+specified any password or additional authentications factors. You must issue a
+password reset link for the users to finish their Teleport registration and be
+able to log in Teleport.
+
+User reset links contain single-use expiring tokens. Because of this, you cannot
+follow the same descriptive approach as for other Teleport resources and
+generate them via a manifest. You need to create those tokens once after the
+user creation, and securely send them to the end-user for them to register their
+password/MFA.
+
+<Tabs>
```

Comment:
I would remove the `<Tab(Item)>` tags and incorporate all of these instructions into the body text so users will see both options. This is because a user may not necessarily know whether they want to use `tctl` or build automation until they have read all of the instructions here.